requirements.txt

```diff
@@ -26,4 +26,4 @@ PyLTI1p3==1.12.1
 canvasapi==3.3.0
 
 # Not in pypi https://github.com/Harvard-University-iCommons/django-canvas-oauth
-https://github.com/Harvard-University-iCommons/django-canvas-oauth/archive/v1.1.0.tar.gz
+https://github.com/Harvard-University-iCommons/django-canvas-oauth/archive/v1.1.1.tar.gz
```

Comment:
This package support reference added only  Python 3.6 and Django > 2.0.  https://github.com/Harvard-University-iCommons/django-canvas-oauth/blob/master/setup.py

I guess the support range is large, the pkg got a recent update in 2024. When I did not update to latest I saw this error "TypeError: get_random_string() missing 1 required positional argument: 'length'' 

Looking at the latest commit from this repo they fixed exactly this https://github.com/Harvard-University-iCommons/django-canvas-oauth/commit/299997426b3f9ca830e7d3085f70aba6fc92504c

SO I guess as long as it build/runs it should be ok with the Python/Django